modules/sharing-registry/sharing-data-migrator/src/main/java/org/apache/airavata/sharing/registry/migrator/airavata/AiravataDataMigrator.java

```diff
@@ -205,17 +218,56 @@ public static void main(String[] args) throws SQLException, ClassNotFoundExcepti
             }
         }
 
+        //Creating the everyone group
+        List<Domain> domainList = sharingClient.getDomains(-1, 0);
+        for (Domain domain : domainList) {
+            CredentialStoreService.Client credentialStoreServiceClient = getCredentialStoreServiceClient();
```

Comment:
You can move this out of the loop, no need to create one each time.

Reply:
credentialStoreServiceClient is now initialized outside the loop.